test/e2e/scenario/telemetry.scenario.ts

```diff
@@ -37,11 +36,41 @@ describe('telemetry', () => {
         window.DD_RUM!.addAction('hop', context as any)
       })
       await flushEvents()
-      expect(serverEvents.telemetry.length).toBe(1)
-      const event = serverEvents.telemetry[0] as TelemetryErrorEvent
+      expect(serverEvents.telemetryErrors.length).toBe(1)
+      const event = serverEvents.telemetryErrors[0]
       expect(event.telemetry.message).toBe('bar')
       expect(event.telemetry.error!.kind).toBe('Error')
       expect(event.telemetry.status).toBe('error')
       serverEvents.empty()
     })
+
+  createTest('send init configuration for logs')
```

Comment:
For `addTelemetryConfiguration` to work, `session`, `viewContexts`, `actionContexts` and `batch` need to be started which is a lot of dependancies.  
I tried to unit test it but it means testing the `startRum` which is hard since we have to be sure to remove all the instrumentation after the test.  
IMO e2e test is a good option, but it has the disadvantage of having to expose `telemetryConfigurationSampleRate`. 
Wdyt?

Reply:
LGTM